_employment/employers/commitments.md

```diff
@@ -21,16 +21,15 @@ template: 1-topic-landing
 
 Employers from every industry and in every state have committed to hiring Veterans and military family members because these individuals have consistently proven themselves to be reliable, independent, and hardworking. 
 
-By signing up and [making a commitment](/_dummy-placeholder.html), your company will stand out to candidates eager to enter the civilian workforce. There are also tax benefits to hiring Veterans, such as the Work Opportunity Tax Credit (WOTC), that range from $1,200 to $9,600. Other VA programs provide incentives to hiring Veterans, such as  [Vocational Rehabilitation and Employment](/_dummy-placeholder.html ) and [On-the-Job Training](/_dummy-placeholder.html). 
+By signing up and [making a commitment](/employment/users/sign_in), your company will stand out to candidates eager to enter the civilian workforce. There are also tax benefits to hiring Veterans, such as the Work Opportunity Tax Credit (WOTC), that range from $1,200 to $9,600. Other VA programs provide incentives to hiring Veterans, such as  [Vocational Rehabilitation and Employment](/_dummy-placeholder.html ) and [On-the-Job Training](/_dummy-placeholder.html). 
```

Comment:
Since they can't make a commitment without signing in, they need to go to
the sign in page first (unless they are already signed in).
On Tue, Mar 29, 2016 at 8:55 PM Molly Blake <notifications@github.com>
wrote:

> In _employment/employers/commitments.md
> <https://github.com/department-of-veterans-affairs/vets-website/pull/1600#discussion_r57834294>
> :
>
> > @@ -21,16 +21,15 @@ template: 1-topic-landing
> >
> >  Employers from every industry and in every state have committed to hiring Veterans and military family members because these individuals have consistently proven themselves to be reliable, independent, and hardworking.
> >
> > -By signing up and [making a commitment](/_dummy-placeholder.html), your company will stand out to candidates eager to enter the civilian workforce. There are also tax benefits to hiring Veterans, such as the Work Opportunity Tax Credit (WOTC), that range from $1,200 to $9,600. Other VA programs provide incentives to hiring Veterans, such as  [Vocational Rehabilitation and Employment](/_dummy-placeholder.html ) and [On-the-Job Training](/_dummy-placeholder.html).
> > +By signing up and [making a commitment](/employment/users/sign_in), your company will stand out to candidates eager to enter the civilian workforce. There are also tax benefits to hiring Veterans, such as the Work Opportunity Tax Credit (WOTC), that range from $1,200 to $9,600. Other VA programs provide incentives to hiring Veterans, such as  [Vocational Rehabilitation and Employment](/_dummy-placeholder.html ) and [On-the-Job Training](/_dummy-placeholder.html).
>
> not sure. let me check with @rcarthur <https://github.com/rcarthur>
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/department-of-veterans-affairs/vets-website/pull/1600/files/ba07e4a5b24cc166d02b3a26467b94d2cb47a479#r57834294>
>